docs/fx-core/frontendhosting-help.md

```diff
@@ -0,0 +1,14 @@
+## FE.StaticWebsiteDisableError
+### Error Message
+Static website hosting feature is disabled for Azure Storage Account.
+### Mitigation
+To host the tab front-end codes in Azure Storage Account, you need to enable the static website feature. You can simply re-run the `Teams - Provision in the Cloud` command and toolkit automatically enable the feature. If you use CLI, try `teamsfx provision`. To manually enable this feature, see [here](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blob-static-website-how-to?tabs=azure-portal#enable-static-website-hosting)
```

Comment:
maybe `code` instead of `codes`?

Reply:
`and toolkit automatically enable the feature`.

1. `the toolkit`
2. `enables` or `will enable`.

Reply:
Fixed.